### ISSUE TITLE
Return loss for validation

### ISSUE THREAD

**Issue by lgvaz** [lgvaz/mantisshrimp_old#1]
Torchvision only calculates loss when model is training, follow [this](https://github.com/pytorch/vision/issues/1350) for more info

**Comment by Lee-JaeSeol** [comment]
fixed it?

**Comment by lgvaz** [comment]
I'm not maintaining this repo anymore, for a more updated library check out [icevision](https://github.com/airctic/IceVision)
